src/compas_timber/connections/l_butt.py

```diff
@@ -79,14 +84,18 @@ def joint_type(self):
         return "L-Butt"
 
     def get_main_cutting_plane(self):
-        angles_faces = beam_side_incidence(self.main_beam, self.cross_beam)
-        cfr = min(angles_faces, key=lambda x: x[0])[1]
+        assert self.main_beam and self.cross_beam
+
+        index, cfr = self.get_face_most_ortho_to_beam(self.main_beam, self.cross_beam, ignore_ends=False)
+        if index in [5, 6]:  # end faces
```

Comment:
my concern is, what happens if someone wants to make a butt joint with beam incidence angle of < 45deg? In this case, the joint will have one of the ends as `cfr` or `index` will be 5 or 6, even though the user wants to join in the old way with just the 4 main faces. 

Should we add this as a kwarg in the GH component? it could be a cutoff angle with default of 45.

Reply:
a sketch here would help. 

but if I get it right though, what you're suggesting sounds like an l-miter, no?
otherwise, when considering only the 4 main faces you would get something like this:
![image](https://github.com/gramaziokohler/compas_timber/assets/3398309/4c7faf93-6127-4f77-830c-7a392a299a10)